src/utils/keys.js

```diff
@@ -108,29 +110,30 @@ export const deriveKeyPairV1 = async (seed, coinObj, channel) => {
 export const deriveKeypairV0 = async (seed, coinObj, channel) => {
   if (channel === DLIGHT_PRIVATE) {
     const spendingKey = await parseDlightSeed(seed)
+    const viewKey = await Tools.deriveViewingKey(seed);
 
     return {
       pubKey: null,
       privKey: spendingKey,
-      viewingKey: await VerusLightClient.deriveViewingKey(spendingKey),
+      viewingKey: viewKey,
```

Comment:
Should discuss if this one needs changed.

I don't know that we use V0 anywhere, but our encoded viewkey is a unified key, now (`uview....` bech32m-encoded).
 
Easiest solution is just to not store the viewkey, since we compute these internally in RN functions that need it, from spendkey or seed.